### PR TITLE
Load default image

### PR DESCRIPTION
**Pull request by khutten**
Issue with multiple images showing up when after second image load and not having default image solve together. Multiple images came from having multiple instances of todaysCuriosity class. A single instance is now made when the default image is loaded, all succeeding images use the original class.

### CODE REVIEW

**Comment by mrhut10** [comment]
Looks like will resolved issue#7
Look forward to walking through it tonight.
Thankn you @khutten

**Comment by khutten** [comment]
Yes it does resolve issue #7 , not sure how to link the two?
https://github.com/mrhut10/todaysCuriosity/pull/11

File: scripts.js [hunk 1]
```diff
@@ -1,32 +1,51 @@
...
+defaultImage.src = george;
+defaultImage.onload = function() {
+  curiosity = new todaysCuriosity(defaultImage);
+  imageSetup(defaultImage);
+};
+
 document.getElementById('file-input').onchange = function (e) {
   loadImage(
       e.target.files[0],
-      function (img) {
```
Comment (khutten) [inline scripts.js]: This function has been moved into `imageSetup`.

**Comment by mrhut10** [comment]
looks great

**Pull request merged**